src/Webfactory/Slimdump/DumpTask.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Webfactory\Slimdump;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Webfactory\Slimdump\Config\Config;
+use Webfactory\Slimdump\Config\ConfigBuilder;
+use Webfactory\Slimdump\Database\Dumper;
+use Doctrine\DBAL\Connection;
+
+final class DumpTask
+{
+    /**
+     * DumpTask constructor.
+     * @param string $dsn
+     * @param string[] $configFiles
+     * @param OutputInterface $output
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function __construct($dsn, $configFiles, OutputInterface $output)
+    {
+        $config = ConfigBuilder::createConfigurationFromConsecutiveFiles($configFiles);
+        $db = \Doctrine\DBAL\DriverManager::getConnection(
+            array('url' => $dsn, 'charset' => 'utf8', 'driverClass' => 'Doctrine\DBAL\Driver\PDOMySql\Driver')
```

Comment:
Use the Driver namespace and ::class constant to improve readability and static meaning.

Reply:
@MalteWunsch I think it's better to keep the 'driverClass' parameter the way it is and to not replace it by Driver::Class because it's not immediately clear what the fallback Driver class is going to be used then (mysql,sqlite, etc.).

If you would import the Driver like this: 
'use Doctrine\DBAL\Driver\PDOMySql\Driver as PdoMySqlDriver' 
which would result in PdoMySqlDriver::class then I would be fine with it.

Reply:
See https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/DriverManager.php#L417
for the purpose of the driverClass parameter.